mantidproject/mantid
# Preliminary support for Panther at ILL

This PR adds preliminary support for the Panther instrument currently under construction at the ILL.

**To test:**

Check that you can load PANTHER using `LoadEmptyInstrument` and is looks alright in the Instrument view.

Request a prototype NeXus data file from me as it is 79MB and I would rather send it over Slack or similar. Use `LoadILLTOF` to load the data. It contains some random counts, but check that the monitor is the last spectrum.

Fixes #25110 and fixes #24805.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
